graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -7,9 +7,10 @@
 )
 import six
 
-from ...schema import (
+from graphql_compiler.schema import (
```

Comment:
Imports within the package need to be relative, please change this to a relative import: `from ..schema import (`